openquake/commonlib/readinput.py

```diff
@@ -1160,16 +1160,18 @@ def get_input_files(oqparam, hazard=False):
     fnames = set()  # files entering in the checksum
     uri = oqparam.shakemap_uri
     if isinstance(uri, dict) and uri:
-        if uri['kind'] == 'shapefile':
-            fname = os.path.join(oqparam.base_path, uri['fname'])
-            fnames.update(get_shapefiles(os.path.dirname(fname)))
-        else:  # xml local files
-            for key, val in uri.items():
-                if key == 'fname' or (
-                        key.endswith('_url') and os.path.exists(val)):
-                    fname = os.path.join(oqparam.base_path, val)
+        # local files
+        for key, val in uri.items():
+            if key == 'fname' or key.endswith('_url'):
+                val = val.replace('file://', '').replace('file:', '')
```

Comment:
I understand 'file://` but not `file:`. I thought `file:` without the `//` was an error.

Reply:
Yeah, its a bit confusing. 

I removed the reference to the file:// solution from the documentation now. Absolute Paths can also be specified now without the "file:". file: denotes still a valid URI therefore I left it in.

- A valid file URI must therefore begin with either file:/path (no hostname), file:///path (empty hostname), or file://hostname/path. If host is omitted, it is taken to be "localhost", the machine from which the URL is being interpreted.

Admittedly my solution is not working when using a Hostname, then again, that would be going a bit far imo.

Reply:
I would support only `file:///path`

Reply:
ok